src/css/page_sections/Menu/_Menu.scss

```diff
@@ -9,13 +9,22 @@
   width: 100vw;
   z-index: 2;
   @include media($coa-large-screen) {
+    background: none;
     bottom: auto;
     left: auto;
     margin: $coa-space-level5 auto 0;
     overflow-y: visible;
-    position: static;
+    position: relative;
     top: auto;
     width: 100%;
+    z-index: 2;
```

Comment:
maybe we need to get more sophisticated with how we do z-index. Maybe this is just an issue to log for later.
https://css-tricks.com/handling-z-index/

Reply:
💅